superset-frontend/src/dashboard/reducers/nativeFilters.ts

```diff
@@ -75,23 +91,41 @@ export function getInitialState({
   return state as NativeFiltersState;
 }
 
+const getUnitState = (
+  unitName: string,
```

Comment:
I think this could be clearerif `unitName` were an enum type, something like `FilterStateType`, after which this function could be renamed `getFilterState` and `unitName` would be just `type`.

Reply:
done